doc/internal/man3/ossl_serial_number_print.pod

```diff
@@ -0,0 +1,35 @@
+=pod
+
+=head1 NAME
+
+ossl_serial_number_print
+- Print a CertificateSerialNumber
+
+=head1 SYNOPSIS
+
+ #include <crypto/x509.h>
+
+ int ossl_serial_number_print(BIO *out, const ASN1_INTEGER* bs, int indent);
+
+=head1 DESCRIPTION
+
+This function writes a certificate serial number I<bs> to BIO I<out> in decimal if the serial
+number can be read into a long int, as well as hexadecimal. It notifies the
+user of negative serial numbers, and handles wrapping of huge serial numbers.
+The output is written to a newline and indented by I<indent> spaces if the
+serial number cannot be read into a long int.
```

Comment:
Also mention that a separating space is prepended if the short format is used.

Reply:
I will still do this if you insist, but I did this because I reasoned that adding a leading space is not intuitive behavior for a function that is named `ossl_serial_number_print`, which I would only expect to print a serial number and nothing else. It also yields control over the spacing / alignment to the calling function. IMO, the accident of printing an unnecessary space after "Serial Number:" is worth having a function that is more intuitive in its behavior.

Reply:
> The function still prints a leading EOL for long numbers

You're right. I'll accept whitespace side-effects as a given, then.